install_config/advanced_ldap_configuration/configuring_form_based_authentication.adoc

```diff
@@ -0,0 +1,104 @@
+= Configuring Form-Based Authentication
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+This topic builds upon
+link:../install_config/advanced_ldap_configuration/ssd_for_ldap_failover.html[Setting
+up SSSD for LDAP Failover] and describes how to set up form-based authentication
+for signing into the {product-title} web console.
+
+[[configuring-form-based-authentication-prepare-a-login]]
+== Prepare a Login Page
+
+The {product-title} upstream repositories have a template for forms. Copy that
+to your authenticating proxy on *_proxy.example.com_*:
+
+====
+----
+# curl -o /var/www/html/login.html \
+    https://raw.githubusercontent.com/openshift/openshift-extras/master/misc/form_auth/login.html
+----
+====
+
+[[configuring-form-based-authentication-apache-install]]
+== Install Another Apache Module
+
+To intercept form-based authentication, install an Apache module:
+
+ ====
+ ----
+ # yum -y install mod_intercept_form_submit
+ ----
+ ====
+
+
+[[configuring-form-based-authentication-apache-configuration]]
+== Apache Configuration
+
+Modify *_/etc/httpd/conf.modules.d/55-intercept_form_submit.conf_* and uncomment
+the *LoadModule* line.
+
+Then, add a new section to your *_openshift-proxy.conf_* inside the
```

Comment:
See https://github.com/openshift/openshift-docs/pull/2160/files/271d0cbdbddac9ded3531d1192d0c4f056c5c5e7#diff-534e8787a6d738379482c2d9db43913aR269

It was created in an earlier step.